packages/scxa-experiment-page/src/results/ontologyIds.json

```diff
@@ -0,0 +1,78 @@
+{
```

Comment:
It is OK for now, but should not we store this in our PostgreSQL DB?
What do you think?

Reply:
Yea or a gist should work too, so curator could modify it directly.